### PR TITLE
PHOENIX-5921 Shorten verification phase fail log line and add a metri…

### PR DESCRIPTION
**Pull request by gokceni**
…c for that

### CODE REVIEW

File: phoenix-core/src/it/java/org/apache/phoenix/monitoring/BasePhoenixMetricsIT.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ void assertMutationMetrics(String tableName, int numRows, Map<String, Map<Metric
             String t = entry.getKey();
             assertEquals("Table name didn't match for mutation metrics", tableName, t);
             Map<MetricType, Long> p = entry.getValue();
-            assertEquals("There should have been four metrics", 4, p.size());
+            assertEquals("There should have been four metrics", 5, p.size());
```
Comment (swaroopak) [inline phoenix-core/src/it/java/org/apache/phoenix/monitoring/BasePhoenixMetricsIT.java:113]: nit: five in the text or can use the number in place of text by assigning it to a variable.

File: phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixMetricsIT.java [hunk 2]
```diff
@@ -329,7 +333,7 @@ public void testMetricsForUpsert() throws Exception {
             String t = entry.getKey();
             assertEquals("Table names didn't match!", tableName, t);
             Map<MetricType, Long> p = entry.getValue();
-            assertEquals("There should have been four metrics", 4, p.size());
+            assertEquals("There should have been four metrics", 5, p.size());
```
Comment (swaroopak) [inline phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixMetricsIT.java:336]: same here

File: phoenix-core/src/main/java/org/apache/phoenix/monitoring/GlobalClientMetrics.java [hunk 3]
```diff
@@ -80,6 +81,7 @@
     GLOBAL_MUTATION_BYTES(MUTATION_BYTES),
     GLOBAL_MUTATION_COMMIT_TIME(MUTATION_COMMIT_TIME),
     GLOBAL_MUTATION_BATCH_FAILED_COUNT(MUTATION_BATCH_FAILED_SIZE),
+    GLOBAL_MUTATION_PHASE_THREE_FAILURE_COUNT(PHASE_THREE_FAILURE_SIZE),
```
Comment (gjacoby126) [inline phoenix-core/src/main/java/org/apache/phoenix/monitoring/GlobalClientMetrics.java:84]: Could we have a more descriptive name than PHASE_THREE_FAILURE_COUNT, which is more intuitive if you don't already know a great deal about the indexing pipeline? GLOBAL_MUTATION_INDEX_COMMIT_FAILURE_COUNT maybe? @kadirozde , what do you think would be a good name?

Comment (kadirozde) [reply phoenix-core/src/main/java/org/apache/phoenix/monitoring/GlobalClientMetrics.java:84]: @gjacoby126,  GLOBAL_MUTATION_INDEX_COMMIT_FAILURE_COUNT sounds good to me.

**Comment by gjacoby126** [comment]
JIRA is resolved, closing PR

**Pull request closed without merging**